recipe/meta.yaml

```
@@ -25,11 +27,12 @@ about:
   home: https://developer.nvidia.com/cuda-toolkit
   license_file: LICENSE.txt
   license: LicenseRef-NVIDIA-End-User-License-Agreement
-  license_url: https://docs.nvidia.com/cuda/eula/index.html
+  #license_url: https://docs.nvidia.com/cuda/eula/index.html
```

Can we include this back in? 
```suggestion
  license_url: https://docs.nvidia.com/cuda/eula/index.html
```

---

In this case, linter complains about having `license_file` and `license_url` fields at the same time.

```
- cuda-version-feedstock/recipe/meta.yaml:/about: license_file_overspecified: Using license_file and license_url is overspecified.
```

What are your suggestions?